lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.82.7",
+  "version": "1.0.0-alpha.1",
```

Comment:
Please undo, we can do an alpha release using scripts and GitHub tagging.

Reply:
I'm happy to back it out however, please note that:
1. This change was done to satisfy the API diff validation. If I don't incremement the version, the changelog.txt gives misleading information. i.e. breaking changes are recorded against the wrong version
2. This change was done using the scripts